telepresence/proxy/deployment.py

```diff
@@ -113,6 +125,19 @@ def existing_deployment_openshift(
     return deployment_arg, run_id
 
 
+def _set_expose_ports(expose, deployment_arg, d_json):
+    deployment, container = _split_deployment_container(deployment_arg)
+    container_to_update = _get_container_name(container, d_json)
+
+    for container in d_json["spec"]["template"]["spec"]["containers"]:
+        if container["name"] == container_to_update:
+            # Merge container ports into the expose list
+            expose.merge_automatic_ports([
+                port["containerPort"] for port in container.get("ports", [])
+                if port["protocol"] == "TCP"
+            ])
```

Comment:
nit: Maybe exit the loop once the container has been found?

Reply:
Fixed https://github.com/telepresenceio/telepresence/pull/1358/commits/277c8eb89d66864119e61cc7fb774445eda37611